src/main/java/edu/kis/vh/nursery/factory/DefaultRhymersFactory.java

```diff
@@ -5,7 +5,8 @@
 import edu.kis.vh.nursery.HanoiRhymer;
 import edu.kis.vh.nursery.factory.Rhymersfactory;
 
-public class DefaultRhymersFactory implements Rhymersfactory {
+//Usunięcie 'public'
```

Comment:
Classes in factory package are necessary. This is a part of stack library public API.